extensions/positron-r/src/provider.ts

```diff
@@ -609,8 +619,17 @@ function userRHeadquarters(): string[] {
 // ad hoc binaries this user wants Positron to know about
 function userRBinaries(): string[] {
 	const config = vscode.workspace.getConfiguration('positron.r');
-	const userBinaries = config.get<string[]>('customBinaries');
-	if (userBinaries && userBinaries.length > 0) {
+	const customBinaries = config.get<string[]>('customBinaries') ?? [];
+	if (customBinaries.length > 0) {
+		const userBinaries = customBinaries
+			.map((item) => untildify(item))
+			.filter((item) => {
+				if (path.isAbsolute(item)) {
+					return true;
+				}
+				LOGGER.info(`R custom binary path ${item} is not absolute...ignoring`);
```

Comment:
Should we verify here that path exists if not, log an appropriate message?

Reply:
Added logging where we do the exists check later in the discovery flow: https://github.com/posit-dev/positron/pull/6621/commits/d5160c5c4799c918c401e2e54bc123e5f360fedc

Added some logging to the `recommendedWorkspaceRuntime()` function as well https://github.com/posit-dev/positron/pull/6621/commits/0a0cd85a0bd76aa648c2e96bc172085f97d30a5a